components/phpunit_bridge.rst

```diff
@@ -0,0 +1,146 @@
+.. index::
+   single: PHPUnitBridge
+   single: Components; PHPUnitBridge
+
+The PHPUnit Bridge
+==================
+
+    The PHPUnit Bridge component provides utilities to report legacy tests, and
+    usage of deprecated code and a helper for time-sensitive tests.
+
+It comes with the following features:
+
+* Forces the tests to use a consistent locale (``C``)
```

Comment:
Can you add blank lines between the items and terminate all but the last item with a semicolon (use a full stop for the last item)? And can you do the same for the other lists?